webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/IsosurfaceService.scala

```diff
@@ -186,7 +187,9 @@ class IsosurfaceService @Inject()(
                                   math.ceil(cuboid.depth / voxelDimensions.z).toInt)
 
     val offset = Vector3D(cuboid.topLeft.x, cuboid.topLeft.y, cuboid.topLeft.z)
-    val scale = Vector3D(cuboid.topLeft.resolution) * request.dataSource.scale.toVector
+    val scale = Vector3D(cuboid.topLeft.resolution) * request.dataSource
+      .map(_.scale.toVector)
+      .getOrElse(Vector3D(1, 1, 1))
```

Comment:
This might be the culprit? When computing isosurfaces for volume tracings, this is used? Didn't read much of the code, but this would might match my assumption mentioned via slack.